all_test.go

```diff
@@ -159,6 +163,59 @@ func hasValidHeader(path string, r io.Reader) (bool, error) {
 	return headerRegex.Match(allBytes), nil
 }
 
+func TestGolangVersion(t *testing.T) {
```

Comment:
Possibly write a comment saying what it's actually testing, namely that the golang version is uniform across the repo? (I was wondering how we worked out which version to assert, and figured it out by the end, but a comment or possibly a rename would make that simpler.)